plugins/modules/zos_mvs_raw.py

```diff
@@ -2454,7 +2461,7 @@ def __init__(
         disposition_abnormal=None,
         block_size=None,
         directory_blocks=None,
-        record_format=None,
+        record_format='FB',
```

Comment:
Why is this change needed? RawDatasetDefinition is used to create a dd_data_set object but the issue is actually dealing with the contents of dd_input. If this change is needed then we also need to change the type in the docstring.

Reply:
My bad.